fenix/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -426,6 +431,14 @@
         android:id="@+id/translationsDialogFragment"
         android:name="org.mozilla.fenix.translations.TranslationsDialogFragment"/>
 
+    <dialog
+        android:id="@+id/translationOptionsDialogFragment"
+        android:name="org.mozilla.fenix.translations.TranslationOptionsDialogFragment" />
+
+    <fragment
```

Comment:
Drive-by 🚗 : Why are is fragment, and two dialogs (the one in this change and the one above) global in our navigation graph? Shouldn't they only be accessible from their current entry points (browser and settings)?

Reply:
Ping @iorgamgabriel @ohall-m 🙂

Reply:
Hi, with this task https://bugzilla.mozilla.org/show_bug.cgi?id=1861058 I'm going to redo all the navigation.